test/natural_order_test.exs

```diff
@@ -6,47 +6,55 @@ defmodule NaturalOrderTest do
   property "natural sort" do
     forall {a, b} <- {numeric_string(), numeric_string()} do
       meets_standard_sort_properties(a, b)
-      |> when_fail(IO.puts("""
-          a: #{a}
-          b: #{b}
-          compare a -> b: #{NaturalOrder.compare(a, b)}
-          compare b -> a: #{NaturalOrder.compare(b, a)}
-      """))
+      |> when_fail(
+        IO.puts("""
+            a: #{a}
+            b: #{b}
+            compare a -> b: #{NaturalOrder.compare(a, b)}
+            compare b -> a: #{NaturalOrder.compare(b, a)}
+        """)
+      )
     end
   end
 
   property "same string except numbers" do
     forall {{a, ai}, {b, bi}} <- string_numbers() do
       meets_standard_sort_properties(a, b) &&
-      case NaturalOrder.compare(a, b) do
-        :eq -> ai == bi
-        :lt -> ai < bi
-        :gt -> ai > bi
-      end
-      |> when_fail(IO.puts("""
-          a: #{a}
-          b: #{b}
-          compare a -> b: #{NaturalOrder.compare(a, b)}
-          compare b -> a: #{NaturalOrder.compare(b, a)}
-      """))
-      |> collect(a)
-      |> collect(ai)
+        case NaturalOrder.compare(a, b) do
+          :eq -> ai == bi
+          :lt -> ai < bi
+          :gt -> ai > bi
+        end
+        |> when_fail(
+          IO.puts("""
+              a: #{a}
+              b: #{b}
+              compare a -> b: #{NaturalOrder.compare(a, b)}
+              compare b -> a: #{NaturalOrder.compare(b, a)}
+          """)
+        )
+        |> collect(a)
+        |> collect(ai)
     end
   end
 
-  def meets_standard_sort_properties(a, b) do
-    upcase_a = String.upcase(a)
-    upcase_b = String.upcase(b)
+  test "shuffled lists must return the same ordered result" do
```

Comment:
never Worked with Proper, so please create a proper test. These two tests just cover a basic case.

Reply:
I can do this later. I'm going to merge this so that you can start using it. I will cut a release too.